src/timing.h

```diff
@@ -55,6 +55,10 @@ JL_DLLEXPORT jl_timing_event_t *_jl_timing_event_create(const char *subsystem, c
 JL_DLLEXPORT void _jl_timing_block_init(char *buf, size_t size, jl_timing_event_t *event);
 JL_DLLEXPORT void _jl_timing_block_start(jl_timing_block_t *cur_block);
 JL_DLLEXPORT void _jl_timing_block_end(jl_timing_block_t *cur_block);
+JL_DLLEXPORT int _jl_tracy_enabled(void);
+JL_DLLEXPORT int _jl_ittapi_enabled(void);
+JL_DLLEXPORT int _jl_nvtx_enabled(void);
```

Comment:
Don't forget to add these to `jl_exported_funcs.inc`

Reply:
Actually, scratch that. DLL-exported, but not re-exported to `libjulia` is probably the right config